docs/application/native/guides/notification/minicontrol.md

```diff
@@ -3,7 +3,7 @@
 
 Minicontrol is a small application view that can be shown on the quick panel or lock screen.
 
-This feature is supported in mobile applications only.
+This feature is supported in mobile and wearable applications only.
 
 The main features of the Minicontrol API include:
 
```

Comment:
As Wearable is also supported for minicontrol,
please add a link to wearable api reference on line 32

From
```
To use the [Minicontrol](../../api/mobile/latest/group__MINICONTROL__LIBRARY.html) API,
```
to 
```
To use the Minicontrol API (in [mobile](../../api/mobile/latest/group__MINICONTROL__LIBRARY.html) and [wearable](../../api/wearable/latest/group__MINICONTROL__LIBRARY.html) applications),
```